EXILED/Exiled.API/Features/Player.cs

```diff
@@ -2070,6 +2054,56 @@ public void Disconnect(string reason = null) =>
         /// </summary>
         public void ResetStamina() => Stamina = StaminaStat.MaxValue;
 
+        /// <summary>
+        /// Sets the scale of a player on the server side.
+        /// </summary>
+        /// <param name="scale">The scale to set.</param>
+        /// <param name="viewers">Who should see the updated scale.</param>
+        public void SetScale(Vector3 scale, IEnumerable<Player> viewers)
+        {
+            if (scale == Scale)
+                return;
+
+            try
+            {
+                ReferenceHub.transform.localScale = scale;
+
+                foreach (Player target in viewers)
+                    Server.SendSpawnMessage?.Invoke(null, new object[] { NetworkIdentity, target.Connection });
+            }
+            catch (Exception exception)
+            {
+                Log.Error($"{nameof(SetScale)} error: {exception}");
+            }
+        }
+
+        /// <summary>
+        /// Sets the scale of the player for other players.
+        /// </summary>
+        /// <param name="fakeScale">The scale to set to.</param>
+        /// <param name="viewers">Who should see the fake scale.</param>
+        public void SetFakeScale(Vector3 fakeScale, IEnumerable<Player> viewers)
```

Comment:
I would make an internal system that handles when players join so they have their fake scale applied. It should be as easy as an event subscriber and saving the fake scale in a dictionary.

If you can do that it would be really useful for all the developers that have trouble manually syncing their fake scale, I have been doing this in cursed mod if you need a reference for it.

Reply:
what if they only want 1 player to see the fake size ?

Reply:
In that case they could use the mirror extensions

Reply:
> I would make an internal system that handles when players join so they have their fake scale applied. It should be as easy as an event subscriber and saving the fake scale in a dictionary.
> 
> If you can do that it would be really useful for all the developers that have trouble manually syncing their fake scale, I have been doing this in cursed mod if you need a reference for it.

Will you do this ? Or no time for it now ?